internal/ethapi/api.go

```diff
@@ -1875,6 +1875,27 @@ func (api *PublicDebugAPI) GetBlockRlp(ctx context.Context, number uint64) (hexu
 	return rlp.EncodeToBytes(block)
 }
 
+// GetRawReceipts retrieves the binary-encoded raw receipts of a single block.
+func (api *PublicDebugAPI) GetRawReceipts(ctx context.Context, blockNrOrHash rpc.BlockNumberOrHash) ([]hexutil.Bytes, error) {
+	block, err := api.b.BlockByNumberOrHash(ctx, blockNrOrHash)
+	if err != nil {
+		return nil, err
+	}
+	receipts, err := api.b.GetReceipts(ctx, block.Hash())
```

Comment:
If `blockHrOrHash` is indeed a hash, then there's no need lookup the block, we can just call `api.b.GetReceipts(ctx, hash)` directly. 
Something along these lines 
 ```golang
	var hash common.Hash
	if h, ok := blockNrOrHash.Hash(); {
		hash = h
	} else{
		block, err := api.b.BlockByNumberOrHash(ctx, blockNrOrHash)
		if err != nil {
			return nil, err
		}
		hash = block.Hash()
	}
	receipts, err := api.b.GetReceipts(ctx, hash)
```